lib/easypost/report.rb

```diff
@@ -5,13 +5,42 @@ class EasyPost::Report < EasyPost::Resource
   # Create a Report.
   def self.create(params = {}, api_key = nil)
     url = "#{self.url}/#{params[:type]}"
+
+    columns = params[:columns] || []
+    additional_columns = params[:additional_columns] || []
+
+    url += '?'
+    columns.each do |column|
+      url += "columns[]=#{column}&"
+    end
+    additional_columns.each do |additional_column|
+      url += "additional_columns[]=#{additional_column}&"
+    end
+
+    params = params.reject { |k, _| [:columns, :additional_columns].include?(k) }
+
     wrapped_params = {}
     wrapped_params[class_name.to_sym] = params
 
     response = EasyPost.make_request(:post, url, api_key, params)
+    response['columns'] = columns
+    response['additional_columns'] = additional_columns
```

Comment:
Unsure how I feel about this - would make the signature of this response look different than what's documented. May not be bad to include...